docs/actions.md

```diff
@@ -198,6 +197,44 @@ Usage:
 Use this utility to create a temporarily action that is immediately invoked. Can be useful in asynchronous processes.
 Check out the [above code block](#examples) for an example.
 
+## Actions and inheritance
+
+Only actions defined **on prototype** can be **overriden** by subclass:
+
+```javascript
+class Parent {
+    // on instance
+    arrowAction = () => {}
+
+    // on prototype
+    action() {}
+
+    constructor() {
+        makeObservable(this, {
+            arrowAction: action
+            action: action,
+        })
+    }
+}
+class Child {
+    // THROWS: TypeError: Cannot redefine property: arrowAction
+    arrowAction = () => {}
+
+    // OK
+    action() {}
+
+    constructor() {
+        super()
+        makeObservable(this, {
+            arrowAction: override,
+            action: override,
+        })
+    }
+}
+```
+
+For more information see [subclassing](observable-state.md#subclassing)
```

Comment:
I'd make sure that you hint forward to action.bound, before people start to autoBind everywhere: "To bind a single action to `this`, `action.bound` can be used instead of arrow functions, for more information see subclassing[link]"